howso/engine/trainee.py

```diff
@@ -1010,20 +963,20 @@ def set_auto_ablation_params(
             The threshold ofr the minimum number of cases at which the model should auto-ablate.
         influence_weight_entropy_threshold : float, default 0.6
             The influence weight entropy quantile that a case must be beneath in order to be trained.
-        exact_prediction_features : Optional[List[str]], optional
+        exact_prediction_features : list of str, optional
             For each of the features specified, will ablate a case if the prediction matches exactly.
-        residual_prediction_features : Optional[List[str]], optional
+        residual_prediction_features : list of str, optional
             For each of the features specified, will ablate a case if
             abs(prediction - case value) / prediction <= feature residual.
-        tolerance_prediction_threshold_map : Optional[Dict[str, Tuple[float, float]]], optional
+        tolerance_prediction_threshold_map : dict of str to tuple of float, optional
```

Comment:
Would prefer "Map of str to tuple of float, optional" and that the method accept any `Mapping` or `MutableMapping` here (depending on if the method modifies it). (throughout)